Makefile

```diff
@@ -46,6 +46,10 @@ lint:
 	ruff check --select I .
 	mkdir -p $(MYPY_CACHE) && mypy --install-types --non-interactive $(PYTHON_FILES) --cache-dir $(MYPY_CACHE) --exclude build/ --exclude pebblo_saferetriever --check-untyped-defs || true
 
+lint-fix:
+	ruff check . --fix
+	ruff format
```

Comment:
The equivalent of this recipe/body is available below under different target names (`format` and `format_diff`). I used these target names as they are used in Langchain: [R64-R66](https://github.com/daxa-ai/pebblo/pull/532/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R64-R66)

Reply:
We can remove the redundant command that only fixes imports ([L:67](https://github.com/daxa-ai/pebblo/pull/532/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R67)) since the previous command fixes all errors, including import-related ones.

Reply:
> We can remove the redundant command that only fixes imports ([L:67](https://github.com/daxa-ai/pebblo/pull/532/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R67)) since the previous command(L:65) fixes all errors, including import-related ones.

Sounds good. Removed the redundant cmd and the new target. Let's stick to the one from Langchain that you added.